internal/log/log.go

```diff
@@ -64,45 +66,61 @@ func CreateLogger(
 	pidPath string,
 	removePIDfunc removePIDfunc,
 	foreground bool,
-	bootstrapLogger zap.Logger,
-) (zap.Logger, error) {
+	bootstrapLogger *Logger,
+) (*Logger, error) {
 
-	output := zap.Output(os.Stdout)
+	var output []string
 	if c.LogSink != "" {
-		sink, err := os.OpenFile(c.LogSink, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0644)
+		_, err := os.OpenFile(c.LogSink, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0644)
```

Comment:
You don't need this code if you add the file path to config.OutputPaths. The `config.Build()` will then end up calling zap.Open() on the file.